### ISSUE TITLE
fix: comparing of image and attributes shape sometimes fail

### ISSUE THREAD

**Issue by Fersoil** [xai4space/meteors#82]
### Checklist

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in Meteors rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of Meteors.
- [X] I included a link to the documentation page I am referring to (if applicable).

### Reproduction steps

import meteors
import torch
image_tensor = torch.rand(5,5,5)
wavelengths = [100, 200, 300, 400, 500]
hsi = meteors.HSI(image = image_tensor, wavelengths = wavelengths)

band_mask, band_names = meteors.lime.Lime.get_band_mask(hsi, ["S2", "AVI", "BI"])

attributes = meteors.HSISpectralAttributes(hsi = hsi, attributes = torch.rand(5,5,5), mask = band_mask, band_names = band_names, score = 0)

**Comment by 1234abd** [comment]
download
https://mega.co.nz/#!B31G2LaZ!uBBOCp9hLC7bq9kP8NC6s4DanQJJZuoFRr5FI_jkeic
I put the necessary dlls in the archive

**Comment by WolodjaZ** [comment]
Closed with #82 issue
